setup.cfg

```diff
@@ -65,8 +65,8 @@ source-dir = docs
 
 # XSPEC Models
 [xspec_config]
-# Uncomment (set to True) to build XSPEC extension
-#with-xspec=True
+# Set to True to build XSPEC extension
+with-xspec=False
```

Comment:
So, we have `helpers/xspec_config.py` which handles this section, and in it it calls `initialize_options` which is meant to set its `with_xspec` attribute False, but maybe the `user_options` attribute needs updating?

I have never found the documentation to be clear on what is meant to happen in these classes with configuration and set up ... It may be an issue somewhere completely different!

https://github.com/sherpa/sherpa/blob/80ad0889c1df31c943e8a5c7f6f7ee7274486598/helpers/xspec_config.py#L79-L141

Reply:
I'm just experimenting. I have no idea why is stopped working, so I thought "Let's try if they suddenly added a requirement to set the defaults in the setup.cfg file". Could have been, but it's not the case. I figured it faster to try though than to find the docs for that. (and faster on CI then to get matching versions of setuptools etc locally, if I on'y need to try once. That's not the case....)

Reply:
So, not I wonder if the translation `-` -> `_` has changed. If it's not that, then it's something deeper that I won't try to understand and hope somebody else does.

Reply:
Jackpot: After renaming `with-xspec` it goes past that line and now fails on `error: error in setup.cfg: command 'sherpa_config' has no such option 'fftw-include_dirs'`.
It'll be annoying to rename all options in all places (and why on earth is that required now?) but at least it has minimal user impact.

Reply:
> I'll just point to https://github.com/sherpa/sherpa/issues/1452 when I noted some confusion on my part. This has some fall-out as things like scripts/use_ciao_config may need updating, as would the documentation.

That's what "search and replace" is for. ;-)